frontend/src/Pages/MainPage.tsx

```diff
@@ -25,10 +25,10 @@ const MainPage: React.FC = () => {
   const navigate = useNavigate();
   useEffect(
     ()=>{
-    if (!token){
-    navigate('/login');
-  }
-},[])
+      if (!token){
```

Comment:
please work on the indentation of the code

Reply:
for future references, use the Navigate React component from react-router-dom instead of the navigate function provided by useNavigate, it's better for such case